ec-europa/eubfr-data-lake
# chore(documentation-theme): update to ECL v2 - noissue

Summary:

- Makes use of ECL v2
- Thus does not download hundreds of unnecessary dependencies any more.
- Fixes search on left navigation
- Removes deprecated components (listing, list)
- Does not use ECL.js any more, as it's not really needed

To test:

- Checkout the branch
- Delete `node_modules` folder
- Install dependencies
- Run `yarn docs:html`
- Check the result `cd dist && npx serve .`
